src/ios/PSPDFKitPlugin.m

```diff
@@ -875,6 +875,26 @@ - (NSString *)imageSaveModeAsJSON {
     return [self enumKeyForValue:imageDocument.imageSaveMode ofType:@"PSPDFImageSaveMode"];
 }
 
+- (void)setCompressionQualityForPSPDFDocumentWithJSON:(NSNumber *)option {
```

Comment:
We should probably have a `VALIDATE_DOCUMENT(_pdfDocument )` call.

Reply:
Since these are just setters and getters, I don't think we need this here. We already gracefully handle invalid documents internally. Also we don't have a `command` for delivering the result.